person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
`id` should be set in the `initialize` constructor which is missing in your case. Kindly set the @id preferably to a random number.